.github/workflows/haskell-ci.yml

```diff
@@ -25,7 +25,7 @@ on:
 jobs:
   linux:
     name: Haskell-CI - Linux - ${{ matrix.compiler }}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
@gallais: If you want to modify this workflow, please do it via the patch https://github.com/agda/agda-stdlib/blob/46daf81a51a6e13a0cae107b3e51790f88da7c5c/.github/haskell-ci.patch as described in https://github.com/agda/agda-stdlib/blob/46daf81a51a6e13a0cae107b3e51790f88da7c5c/HACKING.md#updating-the-haskell-ci-workflow . Also, before patching, regenerate it with the latest `haskell-ci` first which in this particular case would have updated the Ubuntu version.